app/src/main/java/de/westnordost/streetcomplete/quests/camping/AddCampType.kt

```diff
@@ -0,0 +1,42 @@
+package de.westnordost.streetcomplete.quests.camping
+
+import de.westnordost.streetcomplete.R
+import de.westnordost.streetcomplete.data.osm.mapdata.Element
+import de.westnordost.streetcomplete.data.osm.mapdata.MapDataWithGeometry
+import de.westnordost.streetcomplete.data.osm.mapdata.filter
+import de.westnordost.streetcomplete.data.osm.osmquests.OsmFilterQuestType
+import de.westnordost.streetcomplete.data.user.achievements.EditTypeAchievement.OUTDOORS
+import de.westnordost.streetcomplete.osm.Tags
```

Comment:
Add a required import:
```suggestion
import de.westnordost.streetcomplete.osm.Tags
import de.westnordost.streetcomplete.quests.camping.CampType.BACKCOUNTRY
```